packages/x-data-grid-generator/tsconfig.build.json

```diff
@@ -8,7 +8,9 @@
     "noEmit": false,
     "emitDeclarationOnly": true,
     "outDir": "build/esm",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "skipLibCheck": true,
```

Comment:
Without this compilation failed due to not finding `node` types in the `exceljs` lib.
I don't think it's that important to check this validity twice (`typescript`step already checks with `node` types defined).

Reply:
Yes, if I remember correctly, that's the actual issue. 👍 
I think it shouldn't affect our end users, should it?
If they have `node` types in their config, it's going to be fine, if they don't - it's going to error out. 🤔

Reply:
Well, in this case, they would already be getting this error with the current setup. 🤔 
Or am I missing something? 🤷

Reply:
Well yes, they'll be getting this error, but because our own package mandates it as well.

Reply:
Yeah, sadly, we depend on a stale package that is causing multiple issues. 🙈